### PR TITLE
DOCSUP-6073: Documented mapContains, mapKeys, mapValues functions

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #18788

### CODE REVIEW

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 1]
```diff
@@ -112,4 +112,129 @@ Result:
 └──────────────────────────────┴───────────────────────────────────┘
 ```
 
-[Original article](https://clickhouse.tech/docs/en/query_language/functions/tuple-map-functions/) <!--hide-->
+## mapContains {#mapcontains}
+
+Determines  whether `map.keys` contains the `key` parameter.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:117]:
```suggestion
Determines  whether the `Map` contains the `key` parameter.
```

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 2]
```diff
@@ -112,4 +112,129 @@ Result:
...
+**Syntax**
+
+``` sql
+mapContains(map, key)
+```
+
+**Parameters** 
+
+-   `map` — Map. 
+-   `key` — Key. Type matches the type of `map.keys`.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:128]:
```suggestion
-   `key` — Key. Type matches the type of `key` parameter.
```

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 3]
```diff
@@ -112,4 +112,129 @@ Result:
...
+
+**Returned value**
+
+-   `1` if `map.keys` contains `key`, `0` if not.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:132]:
```suggestion
-   `1` if `Map` contains `key`, `0` if not.
```

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 4]
```diff
@@ -112,4 +112,129 @@ Result:
...
+```text
+┌─mapContains(a, 'name')─┐
+│                      1 │
+│                      0 │
+└────────────────────────┘
+```
+
+## mapKeys {#mapkeys}
+
+Returns all the keys from `map` parameter.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:160]:
```suggestion
Returns all keys from the `map` parameter.
```

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 5]
```diff
@@ -112,4 +112,129 @@ Result:
...
+mapKeys(map)
+```
+
+**Parameters**
+
+-   `map` — Map.
+
+**Returned value**
+
+-   Array containing all the keys from `map`.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:174]:
```suggestion
-   Array containing all keys from the `map`.
```

File: docs/en/sql-reference/functions/tuple-map-functions.md [hunk 6]
```diff
@@ -112,4 +112,129 @@ Result:
...
+```text
+┌─mapKeys(a)────────────┐
+│ ['name','age']        │
+│ ['number','position'] │
+└───────────────────────┘
+```
+
+## mapValues {#mapvalues}
+
+Returns all the values from `map` parameter.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/tuple-map-functions.md:201]:
```suggestion
Returns all values from the `map` parameter.
```

File: docs/ru/sql-reference/functions/tuple-map-functions.md [hunk 7]
```diff
@@ -176,4 +176,129 @@ select mapPopulateSeries([1,2,4], [11,22,44], 5) as res, toTypeName(res) as type
...
+mapKeys(map)
+```
+
+**Параметры**
+
+-   `map` — контейнер Map. [Map](../../sql-reference/data-types/map.md).
+
+**Возвращаемое значение**
+
+-   Массив со всеми значениями `map`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/tuple-map-functions.md:279]:
```suggestion
-   Массив со всеми значениями контейнера `map`.
```

File: docs/ru/sql-reference/functions/tuple-map-functions.md [hunk 8]
```diff
@@ -176,4 +176,129 @@ select mapPopulateSeries([1,2,4], [11,22,44], 5) as res, toTypeName(res) as type
 └──────────────────────────────┴───────────────────────────────────┘
 ```
 
-[Оригинальная статья](https://clickhouse.tech/docs/en/query_language/functions/tuple-map-functions/) <!--hide-->
+## mapContains {#mapcontains}
+
+Определяет, включает ли в себя `map` параметр `key`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/tuple-map-functions.md:181]:
```suggestion
Определяет, содержит ли контейнер `map` ключ `key`.
```

File: docs/ru/sql-reference/functions/tuple-map-functions.md [hunk 9]
```diff
@@ -176,4 +176,129 @@ select mapPopulateSeries([1,2,4], [11,22,44], 5) as res, toTypeName(res) as type
...
+mapKeys(map)
+```
+
+**Параметры**
+
+-   `map` — контейнер Map. [Map](../../sql-reference/data-types/map.md).
+
+**Возвращаемое значение**
+
+-   Массив со всеми ключами  контейнера `map`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/tuple-map-functions.md:238]:
```suggestion
-   Массив со всеми ключами контейнера `map`.
```

**Comment by olgarev** [comment]
@nikitamikhaylov  Will you review this PR, please?

**Comment by nikitamikhaylov** [comment]
> @nikitamikhaylov  Will you review this PR, please?

Ok.

**Review by nikitamikhaylov — APPROVED** [review]

**Pull request merged**